Scripts/check-quality.sh

```diff
@@ -6,12 +6,12 @@ echo "... checking Swift code..."
 if [ $# -eq 0 ]; then
   swiftlint --quiet --strict
 elif [[ "$1" == "only-changes" ]]; then
-  git diff --staged --name-only | grep ".swift$" | xargs swiftlint lint --quiet --strict
+  git diff --staged --name-only | grep ".swift$" | xargs -I {} swiftlint lint --quiet --strict "{}"
```

Comment:
```suggestion
  git diff --staged --name-only | grep ".swift$" | xargs -I #FILE# swiftlint lint --quiet --strict "#FILE#"
```

Is it possible to use something like the predicate `#FILE#` to be more explicit? Or something else?

Reply:
For mobile devs, nice input. Bash is so explicit 🥇 

> [!NOTE]
> The `#` if for `.xcconfig` maybe. In bash, it's a comment.

Fixed https://github.com/SRGSSR/playsrg-apple/pull/511/commits/a59a169a84fdb49016efa9d1842461c14637da6b.